griptape/drivers/vector/astra_db_vector_store_driver.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import annotations
+
+import logging
+import warnings
+from typing import TYPE_CHECKING, Any, Optional
+
+from attrs import define, field
+
+from griptape.drivers import BaseVectorStoreDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from astrapy import Collection
+    from astrapy.authentication import TokenProvider
+
+GRIPTAPE_VERSION: Optional[str]
+try:
+    from importlib import metadata
+
+    GRIPTAPE_VERSION = metadata.version("griptape")
+except Exception:
+    GRIPTAPE_VERSION = None
+
+logging.basicConfig(level=logging.WARNING)
```

Comment:
Let's remove this, a specific Vector Store Driver shouldn't change the logging config.

Reply:
Agree, I had inadvertently left it there from some early script. Apologies.